#!/usr/bin/env python
PACKAGE="zed_filter"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("h_min", double_t, 0, "Low end of the hue portion of the filter", 0, 0, 255)
gen.add("h_max", double_t, 0, "High end of the hue portion of the filter", 255, 0, 255)
gen.add("s_min", double_t, 0, "Low end of the saturation portion of the filter", 0, 0, 255)
gen.add("s_max", double_t, 0, "High end of the saturation portion of the filter", 255, 0, 255)
gen.add("v_min", double_t, 0, "Low end of the value portion of the filter", 0, 0, 255)
gen.add("v_max", double_t, 0, "High end of the value portion of the filter", 255, 0, 255)

exit(gen.generate(PACKAGE, "zed_filter", "ZedHSVFilter"))
